ndcli/dimcli/__init__.py

```diff
@@ -1144,6 +1144,17 @@ def modify_pool_set_owning_user_group(self, args):
         '''
         self.client.ippool_set_owner(args.poolname, args.group)
 
+    @cmd.register('modify pool set layer3domain',
+            layer3domain_arg,
+            help='set new layer3domain for POOLNAME')
+    def modify_pool_set_layer3domain(self, args):
+        '''
+        Changes the layer3domain of a pool and all its dependencies.
+        '''
+        result = self.client.ippool_set_layer3domain(args.poolname, args.layer3domain)
```

Comment:
This does not use the optional `dry-run` parameter, should be included like in the other ndcli/dimcli functions that call `@updating` functions of the DIM API.
Should be something like:
```
options = OptionDict()
options.set_if(dryrun=args.dryrun)
result = self.client.ippool_set_layer3domain(args.poolname, args.layer3domain, **options)
```

Reply:
This is now added